src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1274,6 +1338,27 @@ private:
     lines.add(kj::str(""));
   }
 
+  static bool checkIsHttpToken(kj::StringPtr val) {
+    // Allowable values for HTTP tokens per RFC 7230 ("token").
+    // Same as valid header field names ("field-name").
+    // Notably, this excludes CR and LF, thus preventing header injection.
+    static const std::regex tokenRe = std::regex("^[a-zA-Z0-9_!#$%&'*+.^`|~-]+$");
+    return std::regex_match(val.cStr(), tokenRe);
+  }
+
+  static bool checkIsHeaderValue(kj::StringPtr val) {
+    static const std::regex headerValRe = std::regex(
+      "^[\x21-\x7E,\x80-\xFF]+([\x20,\x09][\x21-\x7E,\x80-\xFF]+)*?$");
```

Comment:
The spec defines field-content as:

    field-content = field-vchar [ 1*( SP / HTAB ) field-vchar ]

I think the `1*` means "one or more", but your regex appears to interpret it as "exactly one".